src/interface/console.c

```diff
@@ -737,7 +737,17 @@ static int cc_set(const utf8 **argv, int argc)
 		}
 
 		if (strcmp(argv[0], "money") == 0 && invalidArguments(&invalidArgs, double_valid[0])) {
-			gCashEncrypted = ENCRYPT_MONEY(MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100));
+			money32 money = MONEY((int)double_val[0], ((int)(double_val[0] * 100)) % 100);
+			if (gCashEncrypted != ENCRYPT_MONEY(money)) {
+				if (game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETMONEY, money, GAME_COMMAND_CHEAT, 0, 0) != MONEY32_UNDEFINED) {
+					//Change it locally so it shows the accurate value in the 
+					//"console_execute_silent("get money")" line when in network client mode
+					gCashEncrypted = ENCRYPT_MONEY(money);
```

Comment:
In a separate PR. Now, again, did you actually test it? Unless IntelOrca or Gymnasiast agrees, I'm keeping this, or unless you test it and can prove, or (if you really dont want to test it), at least give a scenario that it would desync..